Doc/reference/datamodel.rst

```diff
@@ -1013,6 +1013,23 @@ Internal types
       If a code object represents a function, the first item in :attr:`co_consts` is
       the documentation string of the function, or ``None`` if undefined.
 
+      .. method:: codeobject.co_positions()
```

Comment:
I think it worths add a warning about this is for a limited range for now, considering users of this API might be confused about why at some python versions it is limited to 256.

Reply:
Also please note that, the `end_line` and offsets are going to be None if the code object is constructed under -X no_traceback_ranges

Reply:
Noted these two cases in the docs.